Dockerfile

```diff
@@ -1,76 +1,77 @@
-# using rocker r-vers as a base with R 4.3.1
-# https://hub.docker.com/r/rocker/r-ver
-# https://rocker-project.org/images/versioned/r-ver.html
-#
-# sets CRAN repo to use Posit Package Manager to freeze R package versions to
-# those available on 2023-10-30
-# https://packagemanager.posit.co/client/#/repos/2/overview
-# https://packagemanager.posit.co/cran/__linux__/jammy/2023-10-30
-
-# set proper base image
-ARG R_VERS="4.3.1"
-FROM ghcr.io/rmi-pacta/workflow.pacta:main AS base
+FROM docker.io/rocker/r-ver:4.3.1 AS base
 
 # set Docker image labels
 LABEL org.opencontainers.image.source=https://github.com/RMI-PACTA/workflow.pacta.report
-LABEL org.opencontainers.image.description="Docker image to run PACTA"
+LABEL org.opencontainers.image.description="Docker image to create PACTA reports and executive summaries"
 LABEL org.opencontainers.image.licenses=MIT
-LABEL org.opencontainers.image.title=""
-LABEL org.opencontainers.image.revision=""
-LABEL org.opencontainers.image.version=""
-LABEL org.opencontainers.image.vendor=""
-LABEL org.opencontainers.image.base.name=""
-LABEL org.opencontainers.image.ref.name=""
-LABEL org.opencontainers.image.authors=""
-
-# set apt-get to noninteractive mode
-ARG DEBIAN_FRONTEND="noninteractive"
-ARG DEBCONF_NOWARNINGS="yes"
+LABEL org.opencontainers.image.title="workflow.pacta.report"
+LABEL org.opencontainers.image.vendor="RMI"
+LABEL org.opencontainers.image.base.name="ghcr.io/rmi-pacta/workflow.pacta:main"
```

Comment:
nit: probably should match base defined above `docker.io/rocker/r-ver:4.3.1 AS base`